android/expoview/src/main/java/host/exp/exponent/RNObject.java

```diff
@@ -91,6 +89,15 @@ public String version() {
     }
   }
 
+  public static Object versionedEnum(String sdkVersion, String clazz, String value) {
+    try {
+      return new RNObject("com.facebook.react.common.LifecycleState").loadVersion(sdkVersion).rnClass().getDeclaredField("RESUMED").get(null);
```

Comment:
It looks like if we actually used the `clazz` and `value` parameters passed in, the result of calling this method would become much more predictable.